src/test/java/training/article/article.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Create, edit and delete an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+
+    * def token = "Bearer " + login.response.user.token
+
+  Scenario Outline: E2E an article
+
+    * def createArticle = {"article": {"title": '#(<descriptionTitle>)',"description": '#(<description>)',"body": '#(<descriptionbody>)',"tagList": ["string"]}}
+    * def articleSuccessfully = read('training/article/responsesuccessfully.json')
+
+    Given path 'articles'
+    And header Authorization = token
+    And request createArticle
+    When method post
+    Then status <statusCode>
+    And match  response == articleSuccessfully
+    And match response.article.title contains '<descriptionTitle>'
+    And match response.article.description contains '<description>'
+    And match response.article.body contains '<descriptionbody>'
+    And match response.article.author.username contains username
```

Comment:
that is the same creation you use on creation.feature. Please abstract this logic to one place